.travis.yml

```diff
@@ -18,7 +18,9 @@ branches:
   - master
   - develop
 before_install:
-  - git clone -b develop --single-branch https://github.com/LearnLib/automatalib.git /tmp/automatalib-develop
+    # set environment variables "AUTOMATALIB_FORK", and "AUTOMATALIB_BRANCH" in repository
+    # settings (default: LearnLib, and develop),
+  - git clone -b ${AUTOMATALIB_BRANCH:-develop} --single-branch https://github.com/${AUTOMATALIB_FORK:-LearnLib}/automatalib.git /tmp/automatalib-develop
```

Comment:
It resolves the problem of being able to build a forked learnlib with a forked automatalib (my automatalib branch is not named "develop", but master).

Reply:
But did it work? Looking at the [travis build](https://travis-ci.org/LearnLib/learnlib/builds/336133910) of your PR, it seems Travis still cloned the wrong branch, since e.g. the `automata-serialization-etf` module could not be found (and haven't been build by the previous checkout).

We have also changed the travis config lately, to use the deployed snapshot version of our (new) continuous deployment stage. I think this will break any dependent changesets anyway ...

Reply:
Well for PRs it indeed does not work (doesnt need to). But for compiling my own fork of LearnLib it did work: https://travis-ci.org/Meijuh/learnlib/jobs/336133885.
But I recognize you changed this lately. Can you shed some light on what is different now? How can I build my learnlib fork against my own automatalib fork?

Reply:
Oh, ok. I didn't think of this case. This seems to add useful functionality then and doesn't break upstream, so I'm fine with the changes.

The latest change removed the local building of AutomataLib and configured Maven to use the artifacts deployed to the Sonatype Snapshot Repository. In its current state, you wouldn't be able to build custom (forked) versions of AutomataLib. I was in favor of this change, because it allowed us to use caching (& improve build time) without running into [a travis issue](https://github.com/travis-ci/travis-ci/issues/8208#issuecomment-349240845). But I will revert these changes back to using the local build, because I think I found another way to workaround this issue (and therefore make your suggestion applicable again).